cli/agent.go

```diff
@@ -199,8 +201,9 @@ func workspaceAgent() *cobra.Command {
 	}
 
 	cliflag.StringVarP(cmd.Flags(), &auth, "auth", "", "CODER_AGENT_AUTH", "token", "Specify the authentication type to use for the agent")
-	cliflag.BoolVarP(cmd.Flags(), &noReap, "no-reap", "", "", false, "Do not start a process reaper.")
+	cliflag.StringVarP(cmd.Flags(), &logDir, "log-dir", "", "CODER_AGENT_LOG_DIR", os.TempDir(), "Specify the location for the agent log files")
```

Comment:
nit: I'm curious, you didn't need to update generated CLI or golden files?

Reply:
Indeed, the `coder agent` is a hidden command and thus not covered by that test. Although, I do feel we should add it, so I'll do that. Good thing you pointed this out!